src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -1395,6 +1399,85 @@ public FFIMethodAttributes(MethodInfo method, Dictionary<MethodInfo, Attribute[]
             }
         }
 
+        public FFIMethodAttributes(MethodBase method, Dictionary<MethodInfo, Attribute[]> getterAttributes = null)
```

Comment:
Hey @aparajit-pratap does it make sense to have this method temporarily point to the obsoleted one so their implementations stay in sync until one is removed?